src/table.scss

```diff
@@ -598,6 +585,19 @@ $block: #{$fd-namespace}-table;
         }
       }
     }
+
+    .#{$block}__body {
+      .#{$block}__cell {
+        &--checkbox {
+          position: relative;
+        }
+
+        .fd-checkbox + .fd-checkbox__label:empty {
+          top: -0.25rem;
+          position: absolute;
+        }
```

Comment:
You can't modify properties of one component (checkbox in this case) inside another (table). This is violating the self-contained rules.

Reply:
I agree, but in this case, we're adjusting it only for the checkboxes inside the table.

We also don't have any condensed mode for the checkbox, that's why I done it like that.

Reply:
it doesn't matter that it's only for the checkboxes inside table. We have lots of cases like this. You can't modify props of checkbox inside table scss. You can always add additional styling inside checkbox or modifier class inside table. But this way we can't accept it.